server/models/userModel.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+const validator = require("validator");
+
+const userSchema = new mongoose.Schema({
+  firstName: {
```

Comment:
I think because we're storing first and last name in the Profile model, we don't need it here. I like the way you've written this though, maybe @quddd take a look at this while making the Profile model